### PR TITLE
Updated documentation about nightly tests

### PR DESCRIPTION
**Pull request by lebeg**
## Description ##

Updated links to new nightly jobs.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by pinaraws** [comment]
@mxnet-label-bot add[pr-awaiting-review, Test, CI]

**Comment by lebeg** [comment]
@aaronmarkham

File: tests/nightly/README.md [hunk 1]
```diff
@@ -17,16 +17,16 @@
...
 # Nightly Tests for MXNet 
 
-These are some longer running tests that are scheduled to run every night. 
+These are some longer running tests that are scheduled to run every night, for master and for latest release branches. 
 
 ### Description
 There are two Jenkins pipelines that run these tests - 
-1. Tests on the Source code: http://jenkins.mxnet-ci.amazon-ml.com/view/Nightly%20Tests/job/NightlyTests_onSource/
-2. Tests on Built Binaries: http://jenkins.mxnet-ci.amazon-ml.com/view/Nightly%20Tests/job/NightlyTests_onBinaries/
+1. Tests on source code ([link](http://jenkins.mxnet-ci.amazon-ml.com/job/NightlyTests/))
```
Comment (aaronmarkham) [inline tests/nightly/README.md]:
```suggestion
1. [Tests on source code](http://jenkins.mxnet-ci.amazon-ml.com/job/NightlyTests/)
```

File: tests/nightly/README.md [hunk 2]
```diff
@@ -17,16 +17,16 @@
...
+2. Tests on built binaries ([link](http://jenkins.mxnet-ci.amazon-ml.com/job/NightlyTestsForBinaries/))
```
Comment (aaronmarkham) [inline tests/nightly/README.md]:
```suggestion
2. [Tests on built binaries](http://jenkins.mxnet-ci.amazon-ml.com/job/NightlyTestsForBinaries/)
```

File: tests/nightly/README.md [hunk 3]
```diff
@@ -17,16 +17,16 @@
...
 
 ### Adding a new Nightly Test
-Add your test script to the mxnet repo (preferably)in the tests/nightly folder)
-Make sure to describe in a readme or comments the purpose of the test. 
+Add your test script to the mxnet repo tests/nightly folder. Make sure to describe in a readme or
```
Comment (aaronmarkham) [inline tests/nightly/README.md]:
```suggestion
Add your test script to the MXNet repo's `tests/nightly/` folder. Make sure to describe in a readme or in the 
```

**Pull request merged**